_posts/2017-12-21-14-01-netcore-beta1.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title : "14.1 7.x Beta1"
+description: "7.x Beta1 for ASP.NET and ASP.NET Core"
```

Comment:
ASP.NET Classic and ASP.NET Core

Reply:
I checked the ASP.NET docs, the original version is "ASP.NET": https://www.asp.net/get-started